### ISSUE TITLE
constrained_layout (sometimes) clips multiline title/xlabel/ylabel

### ISSUE THREAD

**Issue by pharshalp** [matplotlib/matplotlib#13843]
### Bug report

**Bug summary**
When a plot title/label consists of multiple lines (with some mathematics elements that require more vertical space), constrained_layout seems to not work correctly.

**Code for reproduction**

```python
import matplotlib.pyplot as plt
fig, ax = plt.subplots(figsize=(3, 3), constrained_layout=True)
ax.set_title('First line \n'+r'Second line $\frac{x}{y}$')
ax.set_ylabel('ylabel')
ax.set_xlabel('xlabel')
fig.savefig('constrained_layout_issue.png')
```

**Actual outcome**
![constrained_layout_issue](https://user-images.githubusercontent.com/15175620/55409505-29d31000-5530-11e9-8474-edc54525eb56.png)

For now, I am avoiding this issue by increasing `h_pad` with `plt.rcParams['figure.constrained_layout.h_pad'] = 6/72`. But this doesn't feel right (there is a lot more blank space left at the bottom of the plot).
![constrained_layout_issue](https://user-images.githubusercontent.com/15175620/55409858-c7c6da80-5530-11e9-8ff2-01427d45f4de.png)

**Matplotlib version**

  * Operating system: Mac OS
  * Matplotlib version: 3.0.3
  * Matplotlib backend (`print(matplotlib.get_backend())`):
  * Python version: 3.7.3
  * Jupyter version (if applicable):
  * Other libraries:

**Comment by ImportanceOfBeingErnest** [comment]
This is fixed by either #11502 or #11499. It will not occur in master anymore and hence not in 3.1 once it is released.

**Comment by jklymak** [comment]
Can you try w master?   Pretty sure this is fixed.

**Comment by pharshalp** [comment]
Thanks! yep, can confirm that this is fixed in the 3.1 branch.
